conda-forge.yml

```diff
@@ -6,16 +6,16 @@ build_platform:
   linux_aarch64: linux_64
   linux_ppc64le: linux_64
 conda_build:
+  error_overlinking: true
   pkg_format: '2'
 conda_forge_output_validation: true
 github:
   branch_name: main
   tooling_branch_name: main
 os_version:
-  linux_64: cos6
+  linux_64: cos7
```

Comment:
We already are on CentOS 7 in our Linux builds. This was just holding an old value. So fixed it here